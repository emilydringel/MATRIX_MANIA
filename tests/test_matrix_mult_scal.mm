def void multiply_s(matrix<int> x, int y){
    
    int sizeOfR1 = getRows(x);
    int sizeOfC1= getColumns(x);

    for(int i = 0; i < sizeOfR1; i=i+1){
        for(int j = 0; j < sizeOfC1; j=j+1){
            x[i,j] = x[i,j] * y;
  	}
    }
}


def int main(){
  
    matrix<int> m = [1,2,3;4,5,6];
    int n = 2;  

    multiply_s(m, n);
    printm(m);
}
