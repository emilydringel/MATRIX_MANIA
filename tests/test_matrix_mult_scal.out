[2,8,6;4,14,12]
[12,16,18;18,20,22]